### PR TITLE
- Created a Brand Embassy pipeline with Auth Credentials

### PR DESCRIPTION
**Pull request by avlaropereira**
-Created a brand Embassy pipeline in order to Attach settings configuration in BE perf simulations.
- Created  a customized Brand Embassy GetBERequester.

### CODE REVIEW

File: requesters/getRequester.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ class GetRequester extends HttpRequester {
     }
 
     onProcessResponse (err, data) {
+ //       console.log(JSON.stringify(data));
```
Comment (IvanMoralesCamacho) [inline requesters/getRequester.js]: you must remove the comment code

File: requesters/creator/brandEmbassy/userAttach.js [hunk 2]
```diff
@@ -7,14 +7,16 @@ class BrandEmbassyAttachUser extends HttpRequester {
...
-			url: this.configuration.url.getUsers,
-		}, this.onSendingRequestResponse.bind(this));
+		let users = 1;
+		let rampUpUsers;
+		let pattern = '0000';
+		let phoneNumber = 4006880001;
+
+		console.log( JSON.stringify(this.cache.data.users[0]));
+		// console.log('Tracking Brand Embassy Users');
+		// this.request.get({
```
Comment (IvanMoralesCamacho) [inline requesters/creator/brandEmbassy/userAttach.js]: you must remove the comment code

File: requesters/getRequester.js (hunk 1 above)
Comment (avlaropereira) [reply requesters/getRequester.js]: Done thanks for the observation.

File: requesters/creator/brandEmbassy/userAttach.js (hunk 2 above)
Comment (avlaropereira) [reply requesters/creator/brandEmbassy/userAttach.js]: done ... thanks for the observation,

**Pull request merged**